### ISSUE TITLE
Question Regarding Operational Fine-tuning

### ISSUE THREAD

**Issue by ttdelgadott** [google-deepmind/graphcast#178]
Hi! I was wondering what training configuration was used for fine-tuning - namely, the learning rate and number of training steps. Additionally, I'm curious if accumulated params (e.g. 2t) were fine tuned with ERA5 (like the FGN paper) or HRES like the other parameters.
